docs/how-to/hip_runtime_api/initialization.rst

```diff
@@ -0,0 +1,124 @@
+.. meta::
+   :description: Initialization.
+   :keywords: AMD, ROCm, HIP, initialization
+
+********************************************************************************
+Initialization
+********************************************************************************
+
+Initialization involves setting up the environment and resources needed for GPU
+computation. The following steps are covered with the initialization:
+
+- Initialize the HIP runtime
+
+  This includes loading necessary libraries and setting up internal data
+  structures.
+
+- Querying and set GPU devices
+
+  Identifying and querying the available GPU devices on the system.
+
+- Setting up contexts
+
+  Creating contexts for each GPU device, which are essential for managing
+  resources and executing kernels. For further details, check :ref:`context
+  section <context_driver_api>`.
+
+Initialize the HIP runtime
+================================================================================
+
+The HIP runtime is initialized automatically when the first HIP API call is
+made. However, you can explicitly initialize it using :cpp:func:`hipInit`
+function, to be able to control the timing of the initialization. The explicit
+initialization example code:
+
+.. code-block:: cpp
+
+  #include <hip/hip_runtime.h>
+  
+  hipError_t err = hipInit(0);
+  if (err != hipSuccess)
+  {
+      // Handle error
+  }
+
```

Comment:
This example code doesn't really add much.

Reply:
Removed.